lints/lint_distribution_point_missing_ldap_or_uri.go

```diff
@@ -7,9 +7,10 @@ When present, DistributionPointName SHOULD include at least one LDAP or HTTP URI
 package lints
 
 import (
+	"strings"
+
```

Comment:
Ideally these separate whitespace changes wouldn't be in the same commit but it's not a big deal.

Reply:
Yeah. Sorry about that. My IDE have done it automatically. If you want it in separate commits, it can be done, but would have to close this PR and open a new one.